containers/elm-0.19.0/README.md

```diff
@@ -0,0 +1,51 @@
+# Elm (0.19.0)
+
+## Summary
+
+*Develop Elm based applications. Includes the elm extension & binary
+
+| Metadata | Value |  
+|----------|-------|
+| *Contributors* | The VS Code Team & xWiiLLz |
```

Comment:
No need to add the VS Code Team here since you're the contributor - feel free to take the credit! 😊

Reply:
Great, I've updated it! 😊